src/osd/ECBackend.cc

```diff
@@ -813,7 +813,7 @@ bool ECBackend::_handle_message(
     handle_sub_read(op->op.from, op->op, &(reply->op), _op->pg_trace);
     reply->trace = _op->pg_trace;
     get_parent()->send_message_osd_cluster(
```

Comment:
I think an even better improvement here would be ``op->op->get_connection()->send_message(...)``

Reply:
In ECBackend/ReplicatedBackend, it always use get_parent()->send_message_osd_cluster(). In order to be consistent, still unchanged. But we change PrimaryLogPG::send_message_osd_cluster into inline func to get the same aim.